### PR TITLE
Ports Space Jesus

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony and Kierany9
add: Adds Space Jesus, whom admins can spawn in to hunt down sinners!
/:cl:

The Messiah can be spawned in somewhere in maint at any time by admins and comes equipped with five special spells known as Miracles. 

- Resurrection: Brings a target back from the dead
- Blood to Wine: Gives someone a bunch of wine and causes dizziness
- Parting Waves: Opens nearby airlocks
- Repent: Wipes someone's antagonist after some time

Name will change based on what the Chaplain choses.
Icons by @FurryMcFlurry and rumpelgeist. Idea and a lot of the work by @kierany9

### CODE REVIEW

**Comment by Sarcalogo** [comment]
I see that you got rid of the cancerous blasphemous JoJo reference. Well that's good.

**Comment by tortellinitony** [comment]
https://zippy.gfycat.com/ImpureLegitimateBaldeagle.webm

Space Jesus

File: hippiestation/code/datums/spells/cluwnecurse.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	desc = "This spell dooms the fate of any unlucky soul to the live of a pitiful cluwne, a terrible creature that is hunted for fun."
 	school = "transmutation"
 	charge_type = "recharge"
-	charge_max	= 600
+	charge_max	= 1200
 	charge_counter = 0
```
Comment (GuyonBroadway) [inline hippiestation/code/datums/spells/cluwnecurse.dm]: Whats up with this?

Comment (KayeArray) [reply hippiestation/code/datums/spells/cluwnecurse.dm]: WHAT DID HE MEAN BY THIS?

**Review by GuyonBroadway — APPROVED** [review]

**Pull request merged**